browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
+    'no-extra-parens': 0,
```

Comment:
When we are using JSX, we tend to add the extra parens to make the resulting code more consistently tabbed:

```js
return (
  <App>
    <Button>Click Me</Button>
  </App>
);
```

vs

```js
return <App>
  <Button>Click Me</Button>
</App>;
```

I'll look and see if the React plugin has an option for this very case, but if not I'd prefer to keep it around for ES6/JSX code.